harvester/sources/extraction/edurep.py

```diff
@@ -0,0 +1,286 @@
+        }
+
+    @classmethod
+    def get_language(cls, node):
+        language = node["schema:identifier"]["schema:inLanguage"]
+        if language == "Dutch":
+            return "nl"
+        elif language == "English":
+            return "en"
+        return "unk"
+
+    @classmethod
+    def get_url(cls, node):
+        files = cls.get_files(node)
+        if not files:
+            return node["portalUrl"]
+        return files[0]["url"].strip()
+
+    @classmethod
+    def get_mime_type(cls, node):
+        files = cls.get_files(node)
+        if not files:
+            return
+        return files[0]["mime_type"]
+
+    @classmethod
+    def get_technical_type(cls, node):
+        files = cls.get_files(node)
+        if not files:
+            return
+        technical_type = settings.MIME_TYPE_TO_TECHNICAL_TYPE.get(files[0]["mime_type"], None)
+        if technical_type:
+            return technical_type
+        file_url = files[0]["url"]
+        if not file_url:
+            return
+        mime_type, encoding = guess_type(file_url)
+        return settings.MIME_TYPE_TO_TECHNICAL_TYPE.get(mime_type, "unknown")
+
+    @classmethod
+    def parse_copyright_description(cls, description):
+        if description is None:
+            return
+        url_match = cls.cc_url_regex.match(description)
+        if url_match is None:
+            code_match = cls.cc_code_regex.match(description)
+            return slugify(description.lower()) if code_match else None
+        license = url_match.group("license").lower()
+        if license == "mark":
+            license = "pdm"
+        elif license == "zero":
+            license = "cc0"
+        else:
+            license = "cc-" + license
+        return slugify(f"{license}-{url_match.group('version')}")
+
+    @classmethod
+    def get_copyright(cls, node):
+        copyright = node["@type"]["lom:copyrightAndOtherRestrictions"]
+        if copyright is None:
+            return "yes"
+        if copyright == "yes":
+            copyright = cls.parse_copyright_description(cls.get_copyright_description(node))
+        return copyright or "yes"
+
+    @classmethod
+    def get_copyright_description(cls, node):
+        license = node["schema:license"]
+        if not license:
+            return
+        return license.text.strip() if license else None
+
+    @classmethod
+    def get_from_youtube(cls, node):
+        url = cls.get_url(node)
+        if not url:
+            return False
+        return cls.youtube_regex.match(url) is not None
+
+    @classmethod
+    def get_authors(cls, node):
+        authors = []
+        for person in node["dcterms:creator"]:
+            authors.append({
+                "name": person,
+                "email": None,
+                "external_id": None,
+                "dai": None,
+                "orcid": None,
+                "isni": None
+            })
+        return authors
+
+    @classmethod
+    def get_educational_levels(cls, node):
```

Comment:
There are indeed different versions of what seems to be the same at first glance. The difference is that there are NL-LOM allowed and/or recommended values. These get extracted in the OAI-PMH version and are availble under lom_educational_levels, but are not available in JSON Search as fas as I know. And then there is this educational_level variant where Edurep stores values that they made up themselves. We need to properly clean this, but they are not offering the alternative anymore. So we'll stick to this one.